apps/ui/src/components/ProposalVoteSingleChoice.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+import { Choice, Proposal } from '@/types';
+
+defineProps<{
+  sendingType: Choice | null;
+  proposal: Proposal;
+}>();
+
+const emit = defineEmits<{
+  (e: 'handleVoteClick', value: number);
```

Comment:
```suggestion
  (e: 'voteClick', value: Choice);
```

We tend to call handler function with `handle` prefix, but emit names don't have that prefix.